tests/unit/algorithms/test_grpo.py

```
@@ -1898,3 +1899,317 @@ def test_validate_returns_empty_when_no_dataloader(self):
+        generation_logprobs[2, 1:5] = 0.5
+        # Sequence 3: large error -> avg ≈ 1.764 (above threshold 1.2)
+        # 4 tokens with exp(1.0)≈2.718, 5 tokens with exp(0)=1 -> (4*2.718+5)/9≈1.764
+        generation_logprobs[3, 1:5] = 1.0
+
+        train_data = self._create_train_data(
+            batch_size, seq_length, prev_logprobs, generation_logprobs
+        )
+        rewards = torch.tensor([1.0, 0.0, 1.0, 0.0])
+
+        # Use threshold 1.2 which should mask sequences 2 and 3
+        max_error, num_masked, masked_pct = compute_and_apply_seq_logprob_error_masking(
```

Ruff (0.14.14)

Unpacked variable `max_error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)